readium/navigator/src/main/java/org/readium/r2/navigator/epub/EpubNavigatorFragment.kt

```diff
@@ -221,7 +216,9 @@ class EpubNavigatorFragment private constructor(
     private lateinit var resourcesSingle: List<PageResource>
     private lateinit var resourcesDouble: List<PageResource>
 
-    lateinit var preferences: SharedPreferences
```

Comment:
The `EpubNavigatorFragment` is in "legacy mode" (with `SharedPreferences` settings) if a `baseUrl` is provided. If it is `null`, we switch to the new Settings API.